shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -2081,4 +2081,15 @@ public boolean isFailure() {
         }
     }
 
+    @Data
+    public static final class Flush implements WireCommand {
```

Comment:
Can we make sure this doesn't go over the wire. Should we make `WireCommandType.FLUSH_SEGMENT` a negative value.

If this makes it over the wire by accident, there won't be a processor for it on the server side and we'll start logging errors.

For redundancy, I'd also override `mustLog()` and return `false`, which will guarantee that even if it does somehow make it on the server side, it won't pollute the logs with it.

Reply:
ok, I will make this WireCommandType.FLUSH_SEGMENT as negative value.
But, writefields of the flush throws an IOExcpetion : https://github.com/pravega/pravega/pull/3953/files#diff-599b4d30bd4eb53895a9c32146254bb0R2092

so, there is no way that it goes on wire. so, it never be on server side.

and readFrom is null see here : https://github.com/pravega/pravega/pull/3953/files#diff-a3019538349210904685160f5be806d5R107;  AppenDecoder will throw error in case  (imaginary) it goes to server.

Reply:
@andreipaduroiu , I have changed the WireCommandType.FLUSH_SEGMENT as negative value.